.github/workflows/acceptance-tests-runner.yml

```diff
@@ -361,7 +362,10 @@ jobs:
       - name: Acceptance Tests
         env:
           MONGODB_ATLAS_LAST_VERSION: ${{ needs.get-provider-version.outputs.provider_version }}
-          ACCTEST_PACKAGES: ./internal/service/advancedclustertpf
+          ACCTEST_PACKAGES: |
+            ./internal/service/advancedcluster
+            ./internal/service/advancedclustertpf
+          MONGODB_ATLAS_TPF_ADV_CLUSTER_TESTS: "true"
```

Comment:
nit: prefer to have ACCTEST_PACKAGES as last env var

Reply:
done in 053deb331cb7c3189a7de3ffa47e597ba5721ac1